### PR TITLE
[Giang-TK] Laravel_elequent

### PR DESCRIPTION
**Pull request by TranKienGiang**
Uploading Male-Fashion _ Template - Google Chrome 2021-09-13 16-08-35.mp4…

Hello Word !!

### CODE REVIEW

**Comment by TranKienGiang** [comment]
https://user-images.githubusercontent.com/83223418/133057543-e1fcaf21-0d31-4826-93ed-453e406d374a.mp4

File: tran_kien_giang/laravel_eloquent/Controller/HomeController.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+use Illuminate\Support\Facades\DB;
+use Illuminate\Http\Request;
+class HomeController extends Controller
+{
+    public function home()
+    {
+        $products = DB::table('shop_products')
+        ->orderBy('sale_off', 'desc')
+        ->orderBy('product_name', 'desc')
+        ->paginate (16);
```
Comment (VanGiang) [inline tran_kien_giang/laravel_eloquent/Controller/HomeController.php:16]:
```suggestion
          ->orderBy('sale_off', 'desc')
          ->orderBy('product_name', 'desc')
          ->paginate (16);
```
16 đưa vào config nhé

File: tran_kien_giang/laravel_eloquent/Controller/HomeController.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+use App\Http\Controllers\Controller;
+use App\Http\Controllers\CategoryController;
+use App\Models\Category;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Http\Request;
+class HomeController extends Controller
+{
+    public function home()
+    {
+        $products = DB::table('shop_products')
```
Comment (VanGiang) [inline tran_kien_giang/laravel_eloquent/Controller/HomeController.php:13]: eloquent

File: tran_kien_giang/laravel_eloquent/Controller/HomeController.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        ->orderBy('sale_off', 'desc')
+        ->orderBy('product_name', 'desc')
+        ->paginate (16);
+
+        $categories = Category::where('is_public', config('category.public'))
```
Comment (VanGiang) [inline tran_kien_giang/laravel_eloquent/Controller/HomeController.php:18]: dùng this đi e ơi

File: tran_kien_giang/laravel_eloquent/shop-spad.blade.php [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+                                <div class="card-heading">
+                                    <a data-toggle="collapse" data-target="#collapseOne">Categories</a>
+                                </div>
+                                <div id="collapseOne" class="collapse show" data-parent="#accordionExample">
+                                    <div class="card-body">
+                                        <div class="shop__sidebar__categories">
+                                            <ul class="nice-scroll">
+                                                <li><a href="/shop">All(40)</a></li>
+                                                @foreach($categories as $category)
+                                                <li><a href="{{route('categories.index', ['id' => $category->id])}}">{{$category->name}}({{$category->quantity}})</a></li>
```
Comment (VanGiang) [inline tran_kien_giang/laravel_eloquent/shop-spad.blade.php:24]:
```suggestion
                                                <li><a href="{{ route('categories.index', ['id' => $category->id]) }}">{{ $category->name }}({{ $category->quantity }})</a></li>
```

**Pull request merged**